scripts/start-isolated

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env sh
+
+set -e
+projectRoot="$(a="/$0"; a=${a%/*}; a=${a:-.}; a=${a#/}/; cd "$a/.." || return; pwd)"
+
+# Check whether mine-support network needs creating
+if [ -z "$(docker network ls --filter name=^mine-support$ --format={{.Name}})" ]; then
+  echo "Creating mine-support network"
+  docker network create mine-support
+fi
+
+(
+  cd "${projectRoot}"
+  docker-compose up --force-recreate $@
+)
```

Comment:
Thought I'd left this comment, but seems to have vanished. I get an error running this script
2019-08-21 15:06:23,484 ERROR [org.apache.activemq.artemis.core.server] AMQ224088: Timeout (10 seconds) while handshaking with /172.30.0.4:40949 has occurred.

Reply:
That's due to the `wait-for` pinging the Artemis host on the AMQP port (5672). We previously used the Artemis web dashboard port (8161) which could potentially lead to a false positive if the dashboard starts up faster than the AMQP service. I switched the port we wait for to guarantee that Artemis' AMQP is available before launching a service that depends on it but it turns out that triggers an apparent error.

It seems to really be a warning, indicating that something connected and didn't complete the AMQP handshake: 
https://stackoverflow.com/questions/55949120/activemq-artemis-error-amq224088-timeout-10-seconds-while-handshaking-has-o

I think this needs more thought - triggering errors (or even warnings) in the logs every time we start a service is bad but so testing the wrong port for the service we depend on strikes me as an inadequate solution. It could be that we need a more customised availability test that completes the AMQP handshake.

Reply:
The error doesn't occur on the calculation service for port 5672 or 8161, but occurs for both port 5672 and 8161 here, so it's not related to the initial 'wait-for' check - the message is displayed after the project startup so is most likely in the code.

The messages are being consumed OK, so it's not worth addressing in this PR as it is an existing bug.

A card should be created in the back log to address the issue, preferably by using the same consuming code used in the calculation service.

Reply:
That's odd, I tested port 8161 on my laptop yesterday and didn't get the error. Anyway, I've created a [ticket](https://eaflood.atlassian.net/browse/FPD-426) to investigate.